api/app/db/models/c_haines.py

```diff
@@ -0,0 +1,85 @@
+""" Class models that reflect resources and map to database tables Continuous Haines.
+"""
+import enum
+from sqlalchemy import (Column, Integer,
+                        Sequence, ForeignKey, UniqueConstraint, Enum, Index)
+from sqlalchemy.orm import relationship
+from geoalchemy2 import Geometry
+from app.db.database import Base
+from app.db.models.common import TZTimeStamp
+
+
+class SeverityEnum(enum.Enum):
+    """ Enumerated values for severity
+    """
+    LOW = "<4"
+    MODERATE = "4-8"
+    HIGH = "8-11"
+    EXTREME = ">11"
```

Comment:
8.0 would be high.

So really what's happening here is:
```
    LOW = "c-haines <4"
    MODERATE = "4 <= c-haines < 8"
    HIGH = "8 <= c-haines < 11"
    EXTREME = "11 < c-haines"
```

@andrea-williams  what do you think - what would be a better way to get the message across without the little white lie, and without confusing people?

Reply:
It would be succinct to write `MODERATE = [4,8)` and `HIGH = [8,11)` but I'm not sure if that's too math-y.

Reply:
It's probably fine to leave it as is for now - I was mostly just concerned that it could cause logical problems in the code if it wasn't specified how 8.0 would be handled, but now that I've read the rest of your code I'm satisfied that it won't be an issue on the code side of things.

Reply:
I've lumped this into a ticket for future consideration.